dex/tests/cow_protocol/ethereum/cow_protocol_ethereum_assert_partial_fill.sql

```diff
@@ -1,37 +0,0 @@
--- Given a list of trades, when we look at their partial_fill, based on the type of the trade we expect partial_fill to behave differently
```

Comment:
Don't we want to keep these tests?

Reply:
they were valuable in development phase, but don't really add value moving forward. we don't run the tests any time and it's hyper focused on one tx. we can keep, but feels like a good opportunity to cleanse the project